services/src/tests/integration/registry/create-resources.spec.ts

```diff
@@ -67,7 +67,13 @@ const policy = {
     ],
 };
 
-const baseResourceGroup = {schemas: [], upstreams: [], upstreamClientCredentials: [], policies: []};
+const baseResourceGroup = {
+    schemas: [],
+    upstreams: [],
+    upstreamClientCredentials: [],
+    policies: [],
+    policyAttachments: {},
```

Comment:
Shouldn't `policyAttachments` not be here?

Reply:
It is technically there because of the way we currently manage the bucket mocking, but I'll remove it to be in line with the tests in `update-resources.spec` (since this is not something we really test here)